### ISSUE TITLE
Build from source code failed for Postgres 11

### ISSUE THREAD

**Issue by ykostsik** [2ndQuadrant/pglogical#256]
- OS: debian:9 (stretch)
- postgres server version: 11.5
- ENV: docker
- pglogical version: 2.2.2

It has become impossible to install `pglogical` from source code after some changes. I suspect that could be caused  by `postgresql-server-dev-11.8` as a dependency. Previous versions of the `dev` package didn't produce any errors. I get the error, when running `make`
```pglogical_dependency.c:235:13: error: static declaration of ‘AcquireDeletionLock’ follows non-static declaration
 static void AcquireDeletionLock(const ObjectAddress *object, int flags);
             ^~~~~~~~~~~~~~~~~~~
In file included from pglogical_dependency.c:29:0:
/usr/include/postgresql/11/server/catalog/dependency.h:198:13: note: previous declaration of ‘AcquireDeletionLock’ was here
 extern void AcquireDeletionLock(const ObjectAddress *object, int flags);
             ^~~~~~~~~~~~~~~~~~~
pglogical_dependency.c:236:13: error: static declaration of ‘ReleaseDeletionLock’ follows non-static declaration
 static void ReleaseDeletionLock(const ObjectAddress *object);
             ^~~~~~~~~~~~~~~~~~~
In file included from pglogical_dependency.c:29:0:
/usr/include/postgresql/11/server/catalog/dependency.h:200:13: note: previous declaration of ‘ReleaseDeletionLock’ was here
 extern void ReleaseDeletionLock(const ObjectAddress *object);
             ^~~~~~~~~~~~~~~~~~~
<builtin>: recipe for target 'pglogical_dependency.o' failed
make: *** [pglogical_dependency.o] Error 1```

**Comment by petere** [comment]
Yes, this was broken in the last minor releases of PostgreSQL. We'll publish an update of pglogical soon.

**Comment by ykostsik** [comment]
Any updates there? thanks

**Comment by petere** [comment]
This has been fixed in release 2.3.2.
